### DISCUSSION TITLE
Drafts cannot be edited/deleted when CPT Default Status is set to "Published"

### DISCUSSION THREAD

**Discussion by chilledfresh** [pods-framework/pods#6839] [Q&A]
I have a custom post type with the Default Status (under Advanced Options tab) set to "Published."

When creating a new entry, if I enter a title and allow the slug to be auto-generated, then navigate away from the edit page without saving, the entry will be saved as a draft. However, the unsaved entry does not show up in the drafts list.

The number of "ghost drafts" in the database does show correctly in the drafts link.
Example: All (10) | Published (8) | Drafts (2)

Ideally, a pod with a default status set to "published" shouldn't have any auto-generated draft entries, since a published entry by definition must also be saved. But if a draft must be created anyway due to WP's native behavior, perhaps Pods could check for auto-generated drafts and automatically delete them from the database. At the very least, any auto-generated drafts should be deletable from within the WP admin area without having to manually remove them from the SQL database.

**Comment by Ramoonus** [comment]
which version of wordpress are you using?
which version of Pods are you using?

**Comment by chilledfresh** [comment]
@Ramoonus I am running the latest versions of each (WP 4.2.4, Pods 2.5.4)

**Comment by quasel** [comment]
Seems the Option to choose the default post status has no effect, which makes perfectly sense if you think about how this internally work's. Maybe this feature needs a rework/rethink

@chilledfresh 
for me the Draft's just show up like expected - i can't see any flaw besides the options just not working ^^

**Comment by sc0ttkclark** [comment]
The option is only used when using $pod->form() for a post type, which isn't used by default since post types get the normal post editor in the admin area. We should make that a bit more clear in the text.
